config.go

```diff
@@ -23,6 +23,22 @@ type Config struct {
 	MinDiskSpace uint64
 	Motion       MotionConfig
 	LEDs         LEDsConfig
+	Turret       TurretConfig
+}
+
+type ServoConfig struct {
+	Active   bool    `yaml:"active"`
+	MinAng   float64 `yaml:"min-ang"`
+	MaxAng   float64 `yaml:"max-ang"`
+	StartAng float64 `yaml:"start-ang"`
+	Pin      string  `yaml:"pin"`
+}
+
+type TurretConfig struct {
+	Active bool        `yaml:"active"`
+	PID    []float64   `yaml:"pid"`
```

Comment:
Are you allowing for PID control to support servos that give feedback?

Reply:
The servos does use a PID using the angle to the target recorded by the camera as the feedback.